src/test/java/com/google/step/youtube/DescriptionParserTest.java

```diff
@@ -31,6 +32,13 @@ public void parseByCompany_0CompanyName1Promocode() {
         assertThat(DescriptionParser.parseByCompany(company, desc), equalTo(Collections.emptyList()));
     }
 
+    @Test
+    public void parseByCompany_1CompanyName0Promocodes() {
+        String company = "Postmates";
```

Comment:
We should abstract that to COMPANY_NAME for more general tests

Reply:
We should try to remove all real examples from our code.

Reply:
Extracted out the company names in parseByCompany() tests.